alf/examples/ddpg_pendulum.gin

```diff
@@ -0,0 +1,42 @@
+
+import alf.algorithms.ddpg_algorithm
+
+
+
+# environment config
+NUM_PARALLEL_ENVIRONMENTS=10
+create_environment.env_name="Pendulum-v0"
+create_environment.num_parallel_environments=%NUM_PARALLEL_ENVIRONMENTS
+
+# algorithm config
+create_ddpg_algorithm.actor_fc_layers=(100, 100)
+create_ddpg_algorithm.critic_fc_layers=(100, 100)
+create_ddpg_algorithm.actor_learning_rate=1e-4
+create_ddpg_algorithm.critic_learning_rate=1e-3
+create_ddpg_algorithm.actor_fc_layers=(400,300)
+CriticNetwork.observation_fc_layer_params=(400,)
+create_ddpg_algorithm.critic_fc_layers=(300,)
+DdpgAlgorithm.target_update_period=5
+OneStepTDLoss.td_error_loss_fn=@losses.element_wise_huber_loss
+
+# training config
+off_policy_trainer.train.initial_collect_steps=1000
+off_policy_trainer.train.mini_batch_length=2
+off_policy_trainer.train.num_steps_per_iter=%NUM_PARALLEL_ENVIRONMENTS
```

Comment:
At off_policy_trainer.py:96 num_steps_per_iter is changed to at least mini_batch_length*mini_batch_size

Reply:
updated 
 
`num_steps_per_iter` is independent with  `mini_batch_length`  when  training with  sampled transitions  ,  `num_steps_per_iter=num_parallel_envs*per_env_step`